<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScopeFoundry Data Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="treeview_groupBox">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>File System</string>
       </property>
       <layout class="QVBoxLayout" name="file_system_layout">
        <item>
         <widget class="QLineEdit" name="browse_dir_lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browse_dir_browse_pushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QLineEdit" name="file_filter_lineEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="dataview_groupBox">
       <property name="title">
        <string>File</string>
       </property>
       <layout class="QVBoxLayout" name="data_view_layout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="configs_layout">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="1" column="2">
           <widget class="QCheckBox" name="auto_select_checkBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
            <property name="text">
             <string>auto</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="view_label">
            <property name="text">
             <string>View:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3" colspan="2">
           <layout class="QHBoxLayout" name="plugin_buttons_layout">
            <item>
             <widget class="QPushButton" name="log_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Log...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="console_pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Console...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="view_name_comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="file_label">
            <property name="text">
             <string>File:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="data_filename_recycle_pushButton">
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>moves current file to trash. (Delete)</string>
              </property>
              <property name="styleSheet">
               <string notr="true">color:red;
font-weight:bold;</string>
              </property>
              <property name="text">
               <string>ðŸ—‘</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="data_filename_rename_pushButton">
              <property name="minimumSize">
               <size>
                <width>45</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <underline>false</underline>
               </font>
              </property>
              <property name="toolTip">
               <string>renames selected file. (Ctrl + R)</string>
              </property>
              <property name="text">
               <string>rename</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="data_filename_browse_pushButton">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="data_filename_lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
